Source/Core/DolphinQt2/MainWindow.cpp

```diff
@@ -474,7 +482,12 @@ void MainWindow::ScreenShot()
 
 void MainWindow::StartGame(const QString& path)
```

Comment:
Is there a reason for having two `StartGame` functions, one that takes a savestate path and one that doesn't, instead of just having one with a default argument?

Reply:
The signal/slot connections won't work otherwise.

Reply:
Hmm, right, the signature probably won't match for the `connect` calls... What do you think of passing a lambda instead, like this:

    connect(m_fifo_window, &FIFOPlayerWindow::LoadFIFORequested, this, [this](const QString& path) {
      StartGame(path);
    });

IMO, that would be neater as we wouldn't have to define two `StartGame` and two `Play` functions.

Reply:
Sure, that seems like a decent solution. And I agree, the `static_cast` syntax is ugly and also a bit tricky to write.